### PR TITLE
GP2-1518: Great Domestic homepage - markets selection

### PR DESCRIPTION
**Pull request by stevejalim**
This changeset adds in the logic needed to make the 'markets selection' section in the middle of the Domestic Homepage (ported from V1) work.

Screenshot 2021-02-23 at 10 35 23

The frontend markup and JS for this feature was already ported in an earlier ticket, and all that was needed was the data.

The backend re-implements as similar logic as possible to how things worked, overall, in V1.
Adding in this logic added 50 queries to the loading of the homepage (taking it to around 94), so I've added some short (5-minute) cacheing of that 50-queries lookup. This means changes in the totals associated with `IndustryTag`s on market guides (`CountryGuidePage`s) (or the addition of a new `IndustryTag`) will take 5 mins to be reflected in the homepage.

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-1518
- [X] Jira ticket has the correct status.
- [X] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [X] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [ ] Frontend assets have been re-compiled.

### Merging

- [ ] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: core/migrations/0060_merge_0059_migrations.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.19 on 2021-02-23 12:12
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0059_correct_image_relation_for_industrytag'),
+        ('core', '0059_add_quote_block_to_case_study_page'),
```
Comment (webbyfox) [inline core/migrations/0060_merge_0059_migrations.py:10]: Is `0059_add_` new migration, cant see in develop or here?

Comment (webbyfox) [reply core/migrations/0060_merge_0059_migrations.py:10]: Just pulled develop and yeah can see there

**Review by webbyfox — APPROVED** [review]
LGTM

**Review by depsiatwal — APPROVED** [review]
LGTM also

**Pull request merged**